src/Simplex/Chat/Protocol.hs

```diff
@@ -322,6 +326,10 @@ msgContentText = \case
   MCText t -> t
   MCLink {text} -> text
   MCImage {text} -> text
+  MCVoice {text, duration} ->
+    if T.null text then voiceMessage else text <> "; " <> voiceMessage
+    where
+      voiceMessage = T.pack $ "voice message " <> show duration <> "s"
```

Comment:
```suggestion
  MCVoice {text, duration} ->
    msg <> if T.null text then "" else "; " <> text
    where
      msg = T.pack $ "voice message " <> show duration <> "s"
```

Duration can be show as MM:SS (see line 888 in Messages.hs, that function can be moved here)